interface/main/tabs/edit_menu.php

```diff
@@ -82,8 +82,7 @@
     <link href="js/jsoneditor/jsoneditor.css" rel="stylesheet" type="text/css">
     <script src="js/jsoneditor/jsoneditor.js"></script>
 
```

Comment:
Yes.  the comments have caused the github syntax check to spaz.  I had no idea it did much syntax checking, as I have only seen such in this case...where it is wrong.